posthog/temporal/data_imports/pipelines/bigquery/source.py

```diff
@@ -13,6 +13,8 @@
 from posthog.temporal.data_imports.pipelines.pipeline.typings import SourceResponse
 from posthog.warehouse.types import IncrementalFieldType
 
+DEFAULT_TABLE_SIZE_BYTES = 150 * 1024 * 1024  # 150 MB
```

Comment:
Copied this one over from the PostgreSQL source.

Reply:
Could this be moved to `posthog/temporal/data_imports/pipelines/pipeline/consts.py` please? It was only in the postgres source because it was the only thing using it initially and it's much easier to manage having only one version of this